### DISCUSSION TITLE
Cannot import name 'Deepgram' from 'deepgram'

### DISCUSSION THREAD

**Discussion by anthonydatastudio** [deepgram/community#501] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

For months I have been using a Colab notebook to transcribe pre-recorded audios. A few days ago the code no longer works because it cannot invoke the deepgram library. I have tried with anaconda and it shows the same error.

CODE:
!pip install deepgram-sdk
from deepgram import Deepgram 

ERROR:

ImportError: cannot import name 'Deepgram' from 'deepgram' (/Users/anthonycascante/anaconda3/lib/python3.11/site-packages/deepgram/__init__.py)

### If possible, please attach your code or paste it into the text box.

import pandas as pd
from deepgram import Deepgram
import json

# The API key we created in step 3
DEEPGRAM_API_KEY = '7fc95bb1e593cc6d7180a2719943f3e9c9f9244e'

# Replace with your file path and audio mimetype
PATH_TO_FILE = '/content/audio1.m4a'
MIMETYPE = 'audio/mp4'

def main():
    # Initializes the Deepgram SDK
    dg_client = Deepgram(DEEPGRAM_API_KEY)

    with open(PATH_TO_FILE, 'rb') as audio:
        source = {'buffer': audio, 'mimetype': MIMETYPE}
        options = { "punctuate": True,'diarize': True,"mode": "meeting", "language": "es-419", "model": "nova", "utterances":"true","smart_format":"true","numerals":"true"}

        print('Requesting transcript...')
        print('Your file may take up to a couple minutes to process.')

        global response
        response = dg_client.transcription.sync_prerecorded(source, options)
        #print(json.dumps(response, indent=4))

main()

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
Hi @anthonydatastudio 

We just released a major version of the Python SDK `v3`, which means doing a `pip install deepgram-sdk` will grab the latest version of the SDK. As you know, moving between major versions (ie `v2` -> `v3`) will always cause breaking interface changes by [semver](https://semver.org/) definition.

We have updated the README to include semver best-practices when developing projects, applications, etc using the Deepgram SDK (or any consumable library). You can read more about it here:
https://github.com/deepgram/deepgram-python-sdk?tab=readme-ov-file#installation

**Reply by anthonydatastudio** [reply]
Solved. Thank you
